eternum-docs/docs/pages/mechanics/resources/resources.mdx

```diff
@@ -0,0 +1,113 @@
+import ResourceIcon from "../../../../components/ResourceIcon";
+
+# Resources
+
+Eternum features a complex resource system where each resource type serves different purposes in the game economy. All
+resources exist as ERC20 tokens that can be produced, traded, and bridged out of the game.
```

Comment:
not accurate. 

resources of the game can exist as ERC-20 when they are bridged out through the central bank